gdk/aws_clients/S3Client.py

```diff
@@ -27,47 +27,21 @@ def create_bucket(self, bucket, region):
         -------
             None
         """
+        if self.valid_bucket_for_artifacts_exists(bucket, region):
+            logging.info("Not creating an artifacts bucket as it already exists.")
+            return
+
         try:
             if region is None or region == "us-east-1":
                 self.s3_client.create_bucket(Bucket=bucket)
             else:
                 location = {"LocationConstraint": region}
                 self.s3_client.create_bucket(Bucket=bucket, CreateBucketConfiguration=location)
-        except ClientError as exc:
-            if exc.response["Error"]["Code"] == "BucketAlreadyExists":
-                logging.error("Bucket already exists. Please provide a different name for the bucket in the configuration.")
-            elif exc.response["Error"]["Code"] == "BucketAlreadyOwnedByYou":
-                if self.bucket_exists_in_same_region(bucket, region):
-                    logging.info("Not creating an artifacts bucket as it already exists.")
-                    return
-                logging.error("Cannot create the artifacts bucket '%s' as it is already owned by you in other region.", bucket)
-            raise Exception(exc) from exc
         except Exception as exc:
-            print(exc)
             logging.error("Failed to create the bucket '%s' in region '%s'", bucket, region)
```

Comment:
use logging.exception so that it will print the actual exception. Right now we have no idea what went wrong.